### PR TITLE
Add AgentMutelist

### PR DESCRIPTION
**Pull request by Aireil**

### CODE REVIEW

**Review by Haselnussbomber — COMMENTED** [review]
A couple nitpicks 😅

File: FFXIVClientStructs/FFXIV/Client/UI/Agent/AgentMutelist.cs [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+// Client::UI::Agent::AgentMustlist
+//   Client::UI::Agent::AgentInterface
+//     Component::GUI::AtkModuleInterface::AtkEventInterface
+[Agent(AgentId.Mutelist)]
+[GenerateInterop]
+[Inherits<AgentInterface>]
+[StructLayout(LayoutKind.Explicit, Size = 0xD0)]
+public unsafe partial struct AgentMutelist {
+    [FieldOffset(0x68)] public Utf8String SelectedPlayerFullName; // includes cross world icon
+    [FieldOffset(0x58)] public ulong SelectedPlayerAccountId;
```
[the remark this answers is not in the document]
Comment (Aireil) [reply FFXIVClientStructs/FFXIV/Client/UI/Agent/AgentMutelist.cs]: I'm blaming SE's compiler for changing the order in different agents!! 😡

File: FFXIVClientStructs/FFXIV/Client/UI/Agent/AgentMutelist.cs [hunk 2]
```diff
@@ -0,0 +1,15 @@
+﻿using FFXIVClientStructs.FFXIV.Client.System.String;
+
+namespace FFXIVClientStructs.FFXIV.Client.UI.Agent;
+
+// Client::UI::Agent::AgentMustlist
```
[the remark this answers is not in the document]
Comment (Aireil) [reply FFXIVClientStructs/FFXIV/Client/UI/Agent/AgentMutelist.cs]: Oopsie!

**Comment by Haselnussbomber** [comment]
Don't mind the bot, it compares with the main branch. There are no breaking changes here. 🙄

**Pull request merged**